src/capnp_compile.erl

```diff
@@ -414,8 +414,8 @@ generate_decode_envelope_fun() ->
 	{ [RecDef], [FunDef], [] }.
 
 -ast_forms_function(#{name => ast_decode_envelope}).
-	-record(message_ref, {current_offset, current_segment, segments}).
-	decode_envelope(<<RawSegCount:32/little-unsigned-integer, Rest/binary>>) ->
+-record(message_ref, {current_offset, current_segment, segments}).
+decode_envelope(<<RawSegCount:32/little-unsigned-integer, Rest/binary>>) ->
```

Comment:
I'd indented these to indicate they're a part of the `-ast_forms_function()` block.